playground/glob-import/index.html

```diff
@@ -1,8 +1,17 @@
+<h1>Glob import</h1>
+<h2>Normal</h2>
 <pre class="result"></pre>
+<h2>Eager</h2>
 <pre class="result-eager"></pre>
+<h2>node_modules</h2>
 <pre class="result-node_modules"></pre>
+<h2>Raw</h2>
 <pre class="globraw"></pre>
+<h2>Property access</h2>
+<pre class="property-access"></pre>
+<h2>Relative raw</h2>
 <pre class="relative-glob-raw"></pre>
+<h2>Side effect</h2>
```

Comment:
Added some titles so it's easier to debug this in dev. Otherwise it's all JSON on the page.